workers/constructors/Worker/same-origin.html

```diff
@@ -10,14 +10,15 @@
 // not propogate to the window before the tests finish
 setup({allow_uncaught_exception: true});
 
-function testSharedWorkerHelper(t, script) {
+function testWorkerHelper(t, script) {
   try {
-    var worker = new SharedWorker(script, '');
+    var worker = new Worker(script);
```

Comment:
I also noticed that the Worker constructor test was still testing the SharedWorker constructor, probably as a result of copy paste. This meant all of these tests failed in Safari, a UA that doesn't support SharedWorkers.

Reply:
Yikes.   Good catch!